jme3-examples/build.gradle

```diff
@@ -20,12 +20,9 @@ dependencies {
     implementation project(':jme3-jbullet')
     implementation project(':jme3-jogg')
     implementation project(':jme3-lwjgl')
-//    implementation project(':jme3-lwjgl3')
```

Comment:
This commented-out line makes it easier for people to switch between LWJGL v2 and v3, which is something I used to do a lot during beta testing of releases. With a little scripting, that functionality could be moved the gradle.properties file, which might make it easier to find.